manpages/lumina-fm.8

```diff
@@ -0,0 +1,39 @@
+.Dd October 24, 2017
+.Dt LUMINA-FM 8
+.Os 1.3.3
+
+.Sh NAME
+.Nm lumina-fm
+.Nd is a utility used for browsing or otherwise interacting with files on the system.
+
+.Sh SYNOPSIS
+.Nm
+.Op <argument>
+.Op <file path>
+
+.Sh DESCRIPTION
+.Nm
+is the CLI command to open Insight File Manager. From the file manager interface, you can
+maintain the files stored on the system. The utility command can take an optional argument and
+file path. If no argument or file path is specified, the command will open the Insight File Manager
+in the home directory. A specified file path while the interface is already open will simply open a
+new tab in the existing window.
+.Pp
+.Bl -tag -width indent
+.It Ic -new-instance
+This argument opens the Insight File Manager in a new window.
+
+.Sh EXAMPLES
+.Bl -tag -width indent
+.It lumina-fm -new-instance
+Opens the Insight File Manager in a new window.
```

Comment:
If this is the same as above, it probably doesn't do the reader much good as an example?